compiler/lib/src/main/scala/analysis/Analysis.scala

```diff
@@ -73,8 +73,12 @@ case class Analysis(
   tlmPacketSet: Option[TlmPacketSet] = None,
   /** Whether a dictionary is needed in code generation */
   dictionaryNeeded: Boolean = false,
-  /** The set of type symbols used by the dictionary */
-  dictionaryTypeSymbolSet: Set[Symbol] = Set()
+  /** The set of alias type symbols used by the dictionary */
+  dictionaryAliasSymbolSet: Set[Symbol] = Set(),
```

Comment:
I think we should call this TypeSymbolSet instead of AliasSymbolSet. The type definitions happen to be aliases, but other type definitions are possible, and we may want to add them later.